src/Jackett.Common/Definitions/bitbr.yml

```
@@ -135,6 +135,11 @@ search:
       attribute: title
     title:
       text: "{{ if .Result.title_optional }}{{ .Result.title_optional }}{{ else }}{{ .Result.title_default }}{{ end }}"
+      filters:
+        - name: re_replace
+          args:
```

The args syntax is incorrect. refer to https://github.com/Jackett/Jackett/wiki/Definition-format#re_replace-1
It's uncommon for Sonarr/Radarr and other torznab apps to use the language id as a tag
```suggestion
          args: ["(?i)\\bDUAL", "Portuguese English"]
```